CharacterGenerator/CharacterGenerator/characterDeetsViewController.m

```diff
@@ -0,0 +1,65 @@
+//
+//  characterDeetsViewController.m
+//  CharacterGenerator
+//
+//  Created by Henna on 8/2/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "characterDeetsViewController.h"
+
+
+@interface characterDeetsViewController ()
+@property (weak, nonatomic) IBOutlet UILabel *nameLabel;
+@property (weak, nonatomic) IBOutlet UILabel *pushupsLabel;
+@property (weak, nonatomic) IBOutlet UILabel *fightLabel;
+@property (weak, nonatomic) IBOutlet UILabel *beerLabel;
+@property (weak, nonatomic) IBOutlet UILabel *superHeroLabel;
+@property (weak, nonatomic) IBOutlet UIImageView *deetsImage;
+
+@end
+
+@implementation characterDeetsViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.nameLabel.text = self.characterName;
+    self.pushupsLabel.text = [NSString stringWithFormat:@"%d",self.pushupLimit];
+    if(self.canFight){
+        
+        self.fightLabel.text = @"Yes";
+    
+    }
+    else {
+        self.fightLabel.text = @"No";
+    }
+    self.beerLabel.text = [NSString stringWithFormat:@"%d",self.beerLimit];
+    self.superHeroLabel.text = self.superHeroName;
+    self.deetsImage.image = self.deetsImageName;
+
+    
+//    @property (nonatomic) int pushupLimit;
```

Comment:
This file has many lines that can be removed — commented-out code, `didReceiveMemoryWarning`, etc.